Frameworks/UIKit/UIView.mm

```diff
@@ -3481,7 +3503,7 @@ - (void)setGestureRecognizers:(NSArray*)gestures {
     [priv->gestures release];
```

Comment:
thanks; surprising that it compiled...

---
In reply to: [95002606](https://github.com/Microsoft/WinObjC/pull/1643#discussion_r95002606) [](ancestors = 95002606)